Integration test `test_remove_stale_moving_parts/test.py::test_remove_stale_moving_parts_without_zookeeper` is flaky

https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=74945&sha=987be5d530d5dd73ef2ecc126647bc7c63734057&name_0=PR&name_1=Integration+tests+%28tsan%2C+5%2F6%29

```
[gw0] [100%] FAILED test_remove_stale_moving_parts/test.py::test_remove_stale_moving_parts_without_zookeeper 

=================================== FAILURES ===================================
_______________ test_remove_stale_moving_parts_without_zookeeper _______________
[gw0] linux -- Python 3.10.12 /usr/bin/python3

started_cluster = <helpers.cluster.ClickHouseCluster object at 0x7f37095e3f40>

    def test_remove_stale_moving_parts_without_zookeeper(started_cluster):
        ch1.query(f"CREATE DATABASE IF NOT EXISTS {DATABASE_NAME}")
    
        q(
            ch1,
            "CREATE TABLE test_remove ON CLUSTER cluster ( id UInt32 ) ENGINE ReplicatedMergeTree() ORDER BY id;",
        )
    
        table_moving_path = Path(get_table_path(ch1, "test_remove")) / "moving"
    
        q(ch1, "SYSTEM ENABLE FAILPOINT stop_moving_part_before_swap_with_active")
        q(ch1, "INSERT INTO test_remove SELECT number FROM numbers(100);")
        moving_part = "all_0_0_0"
        move_response = ch1.get_query_request(
            sql=f"ALTER TABLE test_remove MOVE PART '{moving_part}' TO DISK 's3'",
            database=DATABASE_NAME,
        )
    
>       wait_part_is_stuck(ch1, table_moving_path, moving_part)

test_remove_stale_moving_parts/test.py:91: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
test_remove_stale_moving_parts/test.py:66: in wait_part_is_stuck
    while exec(node, "ls", table_moving_path).strip() != moving_part:
test_remove_stale_moving_parts/test.py:49: in exec
    return node.exec_in_container(
helpers/cluster.py:4068: in exec_in_container
    return self.cluster.exec_in_container(
helpers/cluster.py:2078: in exec_in_container
    result = subprocess_check_call(
helpers/cluster.py:238: in subprocess_check_call
    return run_and_check(args, detach=detach, nothrow=nothrow, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

args = ['docker', 'exec', 'roottestremovestalemovingparts-gw0-ch1-1', 'bash', '-c', 'ls /var/lib/clickhouse/disks/s3/store/01a/01a21348-04f2-4960-84c1-9c925e599a35/moving']
env = None, shell = False, stdout = -1, stderr = -1, timeout = 300
nothrow = False, detach = False

    def run_and_check(
        args: Union[Sequence[str], str],
        env=None,
        shell=False,
        stdout=subprocess.PIPE,
        stderr=subprocess.PIPE,
        timeout=300,
        nothrow=False,
        detach=False,
    ) -> str:
        if shell:
            if isinstance(args, str):
                shell_args = args
            else:
                shell_args = next(a for a in args)
        else:
            shell_args = " ".join(args)
    
        logging.debug("Command:[%s]", shell_args)
        if detach:
            subprocess.Popen(
                args,
                stdout=subprocess.DEVNULL,
                stderr=subprocess.DEVNULL,
                env=env,
                shell=shell,
            )
            return ""
    
        res = subprocess.run(
            args,
            stdout=stdout,
            stderr=stderr,
            env=env,
            shell=shell,
            timeout=timeout,
            check=False,
        )
        out = res.stdout.decode("utf-8", "ignore")
        err = res.stderr.decode("utf-8", "ignore")
        # check_call(...) from subprocess does not print stderr, so we do it manually
        for outline in out.splitlines():
            logging.debug("Stdout:%s", outline)
        for errline in err.splitlines():
            logging.debug("Stderr:%s", errline)
        if res.returncode != 0:
            logging.debug("Exitcode:%s", res.returncode)
            if env:
                logging.debug("Env:%s", env)
            if not nothrow:
>               raise Exception(
                    f"Command [{shell_args}] return non-zero code {res.returncode}: {res.stderr.decode('utf-8')}"
                )
E               Exception: Command [docker exec roottestremovestalemovingparts-gw0-ch1-1 bash -c ls /var/lib/clickhouse/disks/s3/store/01a/01a21348-04f2-4960-84c1-9c925e599a35/moving] return non-zero code 2: ls: cannot access '/var/lib/clickhouse/disks/s3/store/01a/01a21348-04f2-4960-84c1-9c925e599a35/moving': No such file or directory
```

--- a/tests/integration/test_remove_stale_moving_parts/test.py
+++ b/tests/integration/test_remove_stale_moving_parts/test.py
@@ -51,12 +51,13 @@ def exec(node, cmd, path):
             "bash",
             "-c",
             f"{cmd} {path}",
-        ]
+        ],
+        nothrow=True,
     )
 
 
 def wait_part_is_stuck(node, table_moving_path, moving_part):
-    num_tries = 5
+    num_tries = 10
     while q(node, "SELECT part_name FROM system.moves").strip() != moving_part:
         if num_tries == 0:
             raise Exception("Part has not started to move")

